.eslintrc.cjs

```
@@ -2,22 +2,112 @@ module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
+    ecmaVersion: 2022,
+    // Allows for the parsing of modern ECMAScript features
     sourceType: 'module', // Allows for the use of imports
+    project: './tsconfig.eslint.json',
+    tsconfigRootDir: __dirname,
```

ESLint

[error] '__dirname' is not defined.

(no-undef)